.vib/osclass/goss/osclass.yaml

```
@@ -0,0 +1,35 @@
+command:
+  check-app-version:
+    exec: grep "'version'\s*=>" /opt/bitnami/osclass/oc-content/languages/en_US/index.php
+    exit-status: 0
+    stdout:
+      - /'version'\s*=> '{{ .Env.APP_VERSION }}',/
```

Checking the version is enough (and also, the interesting part for us)

```suggestion
      - "{{ .Env.APP_VERSION }}"
```

---

Done!